format/nexus.py

```diff
@@ -150,6 +150,18 @@ def __call__(self, dset):
                 )
 
 
+def local_visit(nxfile, visitor):
+    """
```

Comment:
Could this be made a google-style docstring (as recommended by the [DIALS contributing guide](https://github.com/dials/dials/blob/master/CONTRIBUTING.md)) 
>Write docstrings. We have a mix of styles at the moment, but new docstrings should try to follow Google-style - it has a good balance between clarity and length.